component_HW1/frontend/src/config/ManageData.jsx

```diff
@@ -0,0 +1,124 @@
+// const urlBasic = "http://localhost:3000/api/";
+const urlBasic = "https://react-p8qv.onrender.com/api/";
+
+
+// const urlAuth = "http://localhost:3000/api/auth/";
+const urlAuth = "https://react-p8qv.onrender.com/api/auth/";
+
+export async function getLogin(formData, path)  {
```

Comment:
bad way bescous is two features not 1
for example register may contain other fields and logic
split to functions login and register